### PR TITLE
[PC-1230] Update ESLOV information in Nicla Voice datasheet

### PR DESCRIPTION
**Pull request by canchebagur**
## What This PR Changes
- This PR updates the ESLOV connector information in the Nicla Voice datasheet.

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md [hunk 1]
```diff
@@ -1,473 +1,490 @@
...
+
+1. VCC_IN (5V input)
+2. INT
+3. SCL
+4. SDA
+5. GND
+
+The manufacturer part number of the ESLOV connector is SM05B-SRSS and its matching receptacle manufacturer part number is SHR-05V-S-B. 
+
+The manufacturer part number of the ESLOV connector is SM05B-SRSS, and its matching receptacle manufacturer part number is SHR-05V-S-B.
```
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md:241]:
```suggestion
```

Duplicated please remove

File: content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md [hunk 2]
```diff
@@ -1,473 +1,490 @@
...
+
+**Notice**: Pin 1 of the ESLOV connector of the Nicla Voice board is a **5V voltage input**, not a voltage output. This means that the Nicla Voice is intended to be used as a secondary or a peripheral board of a main board or controller with an ESLOV connector with a 5V output, for example, the Portenta H7.
+### Sample Sketches
```
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md:244]:
```suggestion

### Sample Sketches
```

adding a break line just for cleaning

File: content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md [hunk 3]
```diff
@@ -1,473 +1,490 @@
...
+2. INT
+3. SCL
+4. SDA
+5. GND
+
+The manufacturer part number of the ESLOV connector is SM05B-SRSS and its matching receptacle manufacturer part number is SHR-05V-S-B. 
+
+The manufacturer part number of the ESLOV connector is SM05B-SRSS, and its matching receptacle manufacturer part number is SHR-05V-S-B.
+
+**Notice**: Pin 1 of the ESLOV connector of the Nicla Voice board is a **5V voltage input**, not a voltage output. This means that the Nicla Voice is intended to be used as a secondary or a peripheral board of a main board or controller with an ESLOV connector with a 5V output, for example, the Portenta H7.
```
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md:243]:
```suggestion
**Note**: Pin 1 of the ESLOV connector of the Nicla Voice board is a **5V voltage input**, not a voltage output. This means that the Nicla Voice is intended to be used as a secondary or a peripheral board of a main board or controller with an ESLOV connector with a 5V output like, for example, the Portenta H7.
```

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Perform the small changes and ready to merge.

**Comment by canchebagur** [comment]
@jcarolinares PDF file updated with the latest changes.

**Comment by canchebagur** [comment]
@jcarolinares done!

**Review by jcarolinares — APPROVED** [review]
Approved and merging

**Pull request merged**